keys/show.go

```diff
@@ -67,11 +71,13 @@ func runShowCmd(cmd *cobra.Command, args []string) (err error) {
 		isOutputSet = tmp.Changed
 	}
 
-	if isShowAddr && isShowPubKey {
+	isShowEitherAddr := isShowAddr || isShowEthAddr
+
+	if isShowEitherAddr && isShowPubKey {
 		return errors.New("cannot use both --address and --pubkey at once")
```

Comment:
Would this not be `--ethwallet` instead of `--address`?

Reply:
Yeah I can update the error